instrumentation/opentelemetry_oban/lib/opentelemetry_oban/job_handler.ex

```diff
@@ -99,6 +100,16 @@ defmodule OpentelemetryOban.JobHandler do
     Span.record_exception(ctx, error, stacktrace)
     Span.set_status(ctx, OpenTelemetry.status(:error, ""))
 
+    set_measurements_attributes(measurements)
+
     OpentelemetryTelemetry.end_telemetry_span(@tracer_id, metadata)
   end
+
+  defp set_measurements_attributes(%{duration: duration, queue_time: queue_time}) do
+    OpenTelemetry.Tracer.set_attributes(%{
+      :"messaging.oban.duration_us" => System.convert_time_unit(duration, :native, :microsecond),
```

Comment:
@tsloughter based on you slack question about not extending semcon namespaces, do you think we should look at merging this and then do a single pass fixing that?

Reply:
Better if we don't add any that are incorrect.

So switching this to `oban.messaging...` is needed.

There really needs to be some semconv for jobs libraries instead of just using messaging. I've seen this same thing in other languages, like Ruby's que.

Reply:
Yeah. There's just other attributes with the issue so didn't want to end up with mixed naming

Reply:
I vote we keep it as is in this PR and normalize all of it in another one prior to cutting a release.